zerver/lib/bugdown/api_arguments_table_generator.py

```diff
@@ -117,11 +117,18 @@ def render_table(self, arguments: List[Dict[str, Any]]) -> List[str]:
 
             # TODO: OpenAPI allows indicating where the argument goes
             # (path, querystring, form data...).  We should document this detail.
+            example = ""
+            if 'example' in argument:
+                example = escape_html(json.dumps(argument['example']))
+            else:
+                content_type = list(argument['content'].keys())[0]
+                example = escape_html(json.dumps(argument['content'][content_type]['example']))
+
             table.append(argument_template.format(
                 argument=argument.get('argument') or argument.get('name'),
                 # Show this as JSON to avoid changing the quoting style, which
                 # may cause problems with JSON encoding.
-                example=escape_html(json.dumps(argument['example'])),
+                example=example,
```

Comment:
Leave the `escape_html` on this line, not the assignments above. Escaping should always happen as close to the output as possible.

Reply:
Edited accordingly.